lib/davinci_dtr_test_kit/client_groups/dinner_static/dtr_dinner_questionnaire_package_request_test.rb

```diff
@@ -11,39 +11,73 @@ class DTRDinnerQuestionnairePackageRequestTest < Inferno::Test
       Inferno will wait for a DTR questionnaire package request from the client. Upon receipt, Inferno will generate and
       send a response.
     )
-    input :smart_app_launch, type: 'radio', title: 'SMART App Launch',
-                             description: 'How will the DTR SMART App launch?',
-                             options: { list_options: [{ label: 'Launch from Inferno', value: 'inferno' },
-                                                       { label: 'Launch from EHR', value: 'ehr' }] }
+    input :smart_app_launch,
+          type: 'radio',
+          title: 'SMART App Launch',
+          description: 'How will the DTR SMART App launch?',
+          options: { list_options: [{ label: 'EHR Launch from Inferno', value: 'ehr' },
```

Comment:
If any presets use this value, they'll need to be updated too.

Reply:
no presets use that input, but this did remind me to update the preset for running the two tests against each other.